### ISSUE TITLE
Cannot use Grafana in older versions of Edge

### ISSUE THREAD

**Issue by 141Mr-liu** [grafana/grafana#58935]
**Steps to reproduce**:
Using Grafana in multiple browsers

**Actual Result**:
Can use Grafana normally.

**Expected Result**
Cannot use Grafana in Edge

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version: 9.2.5
- User OS & Browser: Edge 42.17134.1098.0
- Others: I have tried other versions of Grafana, and the problem still occurs

**Comment by zuchka** [comment]
Thanks for creating this issue, @141Mr-liu 

that's a fairly old browser version. Correct me if I'm wrong here but isn't the newest version of edge v.107? That's 60+ major versions ahead. I don't know if Grafana makes any guarantees for extremely long-term backwards compatibility with legacy browser version.

have you tried upgrading?
